python/HeaderService/camera_coords.py

```diff
@@ -253,62 +291,170 @@ def mosaic_ITL(self, Sx, Sy):
     def get_channel(self, Segment):
         return CHANNEL[self.vendor][Segment]
 
-    def wcs_tan_matrix(self, ra, dec, rot_angle=0):
+    def wcs_TAN(self, ra, dec, rot_angle, radesys):
 
         '''
         Compute the common section of the WCS Tangential projection matrix
         See for a definition http://danmoser.github.io/notes/gai_fits-imgs.html
         '''
 
-        CDELT1 = self.xpixelscale
-        CDELT2 = self.ypixelscale,
-        CROTA2 = rot_angle
         wcs = {'CTYPE1': 'RA---TAN',
                'CTYPE2': 'DEC--TAN',
                'CUNIT1': 'deg',
                'CUNIT2': 'deg',
                'CRVAL1': ra,
                'CRVAL2': dec,
-               # ------------------------------------------
-               # We will use a function for these later on
-               # 'CRPIX1' : 5,
-               # 'CRPIX2' : 10,
-               # ------------------------------------------
-               'CROTA2': rot_angle,
-               'CDELT1': self.xpixelscale,
-               'CDELT2': self.ypixelscale,
-               'CD1_1': CDELT1*math.cos(CROTA2),
-               'CD1_2': -CDELT2*math.sin(CROTA2),
-               'CD2_1': CDELT1*math.sin(CROTA2),
-               'CD2_2': CDELT2*math.cos(CROTA2),
+               'CRPIX1': self.CRPIX1,
+               'CRPIX2': self.CRPIX2,
+               'CD1_1': self.CDELT1*math.cos(rot_angle),
+               'CD1_2': -self.CDELT2*math.sin(rot_angle),
+               'CD2_1': self.CDELT1*math.sin(rot_angle),
+               'CD2_2': self.CDELT2*math.cos(rot_angle),
+               'RADESYS': radesys,
                }
         return wcs
 
-    def wcs_CCD(self, Sx, Sy):
+    def wcs_C(self, Sx, Sy):
+
+        """
+        This function defines the WCS representation for a Segment (Sx,Sy) in
+        CCD coordinates as described in section 9.6 of LCA-13501 for e2V and
+        ITL sensors.
+        """
```

Comment:
LCA-13501 refers to camera-side EOtest, it does not apply to how we will be writing LSST data for science processing.

Reply:
LCA-13501 is relevant here as `ATCamera` is using  this as the reference for the `DTVX/DTMXX` matrix as well as `DATASEC`, `DETSIZE`  and `DETSEC` for each segment. Please let know how you want me to proceed.

Reply:
I don't understand.  What do you mean by, "ATCamera" -- the CCS, in which case I'm confused as this is being written in the "OCS" side?  None of the DM code needs these numbers.

Reply:
The `ATHeaderService` needs these values which are captured from the `ATCamera.imageReadoutParameters`. These are broadcasteded by `ATCamera` (i.e. CCS) and the `ATHeaderService` is agnostic about where is `ATCamera` getting them from.